gems/pending/spec/miq_vm/miq_azure_vm_image_spec.rb

```diff
@@ -0,0 +1,81 @@
+require_relative './miq_vm_instance_methods_shared_examples'
+require 'ostruct'
+require 'azure-armrest'
+require 'MiqVm/miq_azure_vm'
+require 'vcr'
+
+describe MiqAzureVm do
+  before(:all) do
+    @test_env = TestEnvHelper.new(__FILE__)
+    @test_env.vcr_filter
+
+    @client_id  = @test_env[:azure_client_id]
+    @client_key = @test_env[:azure_client_key]
+    @tenant_id  = @test_env[:azure_tenant_id]
+    @image_name = @test_env[:image_name]
+    @image_uri  = @test_env[:image_uri]
+
+    @test_env.ensure_recording_dir_exists
+  end
+
+  before(:each) do |example|
```

Comment:
I like being explicit, especially when there's more than one `before` in the group.